starcounter-include.html

```diff
@@ -12,9 +12,29 @@
 
 <script>
     (function() {
+
+        const CSSCache = new Map();
+        async function fetchStyle(url) {
+            if(CSSCache.get(url)) {
+                return CSSCache.get(url).cloneNode(true);
+            } else {
+                const css = await fetch(url).then(res => res.text());
+                const style = document.createElement('style');
+                style.setAttribute('data-added-by',"starcounter-include");
+                style.setAttribute('data-original-href', url);
+                style.innerHTML = css;
+                CSSCache.set(url, style);
+                return style;
+            }
+        }
+
+        // cache uniform asynchronously
+        fetchStyle(new URL('/sys/uniform.css/uniform.css', window.location).href);
```

Comment:
I think `stracounter-include` should not impose `'/sys/uniform.css/uniform.css'` being loaded. It's
- hidden dependency on uniform.css,
- hidden requirement to keep it in that exact path,

I don't see a reason to force Starcounter include users to use uniform.